tests/testthat/test_infillcrits.R

```diff
@@ -33,7 +33,7 @@ test_that("infill crits", {
       expect_true(!anyMissing(opdf$infill_ei[, c("ei","se","mean")]))
     if (!is.null(opdf$infill_cb)) {
       expect_true(!anyMissing(opdf$infill_cb[, c("se","mean","lambda")]))
-      expect_true(all(opdf$infill_cb$lambda == 2))
+      expect_true(all(opdf$infill_cb$lambda == 1))
```

Comment:
Thanks for noticing, that is actually a pretty big bug :(

in `infillCrit.InfillCritCB` it should be `crit$params$cb.lambda` and not `crit$cb.lambda`

fixing now